model_compression_toolkit/core/common/hessian/trace_hessian_config.py

```diff
@@ -0,0 +1,65 @@
+# Copyright 2023 Sony Semiconductor Israel, Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+from model_compression_toolkit.constants import HESSIAN_OUTPUT_ALPHA, HESSIAN_NUM_ITERATIONS
+
+
+class TraceHessianConfig:
+    """
+    Configuration for Hessian-based information computation, specifically for the trace of the Hessian.
+
+    This class is used to define parameters for the approximation of the trace of the Hessian matrix.
+    It's important to note that this does not compute the actual Hessian but approximates some Hessian-based data.
+    The computation can be computationally heavy and is based on the trace of the Hessian.
+
+    """
+
+    def __init__(self,
```

Comment:
This class is becoming unnecessary if we are going to get rid of the "alpha" parameter.
If it does not used in the whole "fetching"/"caching" mechanisms, then maybe consider to remove it completely and just add there to arguments to the calculator.

Reply:
Is there no way we will need different params in the future? Something the user might want to configure?